src/common/FPSTimer.cpp

```diff
@@ -13,16 +13,16 @@
 
 FPSTimer::FPSTimer()
     : mTotalTime(static_cast<float>(NUM_FRAMES_TO_AVERAGE)),
-      mTimeTable(NUM_FRAMES_TO_AVERAGE, 1.0f),
-      mHistoryFPS(NUM_HISTORY_DATA, 1.0f),
-      mHistoryFrameTime(NUM_HISTORY_DATA, 100.0f),
+      mTimeTable(NUM_FRAMES_TO_AVERAGE, 1.0),
+      mHistoryFPS(NUM_HISTORY_DATA, 1.0),
+      mHistoryFrameTime(NUM_HISTORY_DATA, 100.0),
       mTimeTableCursor(0),
       mRecordFpsFrequencyCursor(0),
-      mInstantaneousFPS(0.0f),
-      mAverageFPS(0.0f)
+      mInstantaneousFPS(0.0),
+      mAverageFPS(0.0)
 {}
 
-void FPSTimer::update(float elapsedTime, float renderingTime, int logCount)
+void FPSTimer::update(double elapsedTime, double renderingTime, int logCount)

```

Comment:
If we can trigger the update every several seconds, that would be more reasonable.